cmd/keycloakb/keycloak_bridge.go

```diff
@@ -303,16 +340,6 @@ func main() {
 		defer closer.Close()
 	}
 
-	// Audit events DB.
-	type CloudtrustDB interface {
```

Comment:
This move is not necessary : this part will be removed when using common-service